src/harness.js

```diff
@@ -672,6 +676,40 @@ class Harness {
   }
 }
 
+/**
+ * Loads and fills a contact form with the appropriate action
+ *
+ * @param {string} contactType Type of contact that should be created
+ * @param {string} action one of 'create' or 'edit'
+ * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
+ */
+const fillContactForm = async (harness, contactType, action, ...answers) => {
```

Comment:
in other functions in this file, seems to use `self` instead of `harness` for the name of `this`